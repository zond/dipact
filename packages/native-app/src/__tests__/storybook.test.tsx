import initStoryshots from "@storybook/addon-storyshots";

import "react-native-gesture-handler/jestSetup";

jest.useFakeTimers();

jest.mock("@react-native-community/async-storage", () => ({
    default: null,
}));

jest.mock("@react-native-community/masked-view", () => ({}));

/* Silence the warning: Animated: `useNativeDriver` is
 * not supported because the native animated module is missing
 */
jest.mock("react-native/Libraries/Animated/NativeAnimatedHelper");

initStoryshots({
	framework: "react-native",
	configPath: "src/storybook",
});
