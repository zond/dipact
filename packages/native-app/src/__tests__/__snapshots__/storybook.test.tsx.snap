// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots components/Switch False 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "row",
      }
    }
  >
    <RCTSwitch
      accessibilityRole="switch"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#fef1da"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "alignSelf": "flex-start",
            "display": "flex",
          },
        ]
      }
      testID="switch"
      thumbTintColor="#FFF"
      tintColor="#AAA"
      value={false}
    />
    <Text />
  </View>
</View>
`;

exports[`Storyshots components/Switch False with label 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "row",
      }
    }
  >
    <RCTSwitch
      accessibilityRole="switch"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#fef1da"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "alignSelf": "flex-start",
            "display": "flex",
          },
        ]
      }
      testID="switch"
      thumbTintColor="#FFF"
      tintColor="#AAA"
      value={false}
    />
    <Text>
      Push notifications
    </Text>
  </View>
</View>
`;

exports[`Storyshots components/Switch True 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "row",
      }
    }
  >
    <RCTSwitch
      accessibilityRole="switch"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#fef1da"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "alignSelf": "flex-start",
            "display": "flex",
          },
        ]
      }
      testID="switch"
      thumbTintColor="#FDE2B5"
      tintColor="#AAA"
      value={true}
    />
    <Text />
  </View>
</View>
`;

exports[`Storyshots components/Switch True with label 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "row",
      }
    }
  >
    <RCTSwitch
      accessibilityRole="switch"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#fef1da"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "alignSelf": "flex-start",
            "display": "flex",
          },
        ]
      }
      testID="switch"
      thumbTintColor="#FDE2B5"
      tintColor="#AAA"
      value={true}
    />
    <Text>
      Push notifications
    </Text>
  </View>
</View>
`;
